genesyscloud/tfexporter/export_common.go

```diff
@@ -219,6 +220,40 @@ func sanitizeE164Number(number string) string {
 	}
 	return number
 }
+func sanitizeRrule(input string) string {
+	attributeRegex := map[string]*regexp.Regexp{
```

Comment:
unit tests for this function could be good to make sure the reg expressions work as expected

Reply:
My bad. I assumed that was for `removeTrailingZeros`